angular.json

```diff
@@ -21,6 +21,7 @@
             "index": "src/index.html",
             "main": "src/main.ts",
             "tsConfig": "tsconfig.app.json",
+            "extractCss": true,
```

Comment:
We could remove `extractCss` from the production configuration, right?

Reply:
Yes, `build.options.extractCss ` is already true, ` build.configurations.production.extractCss` can be removed.